lib/fog/aws/credential_fetcher.rb

```diff
@@ -1,3 +1,7 @@
+# frozen_string_literal: true
+
+require "nokogiri"
```

Comment:
I think this should already be available via `fog-xml`? Might be better to not have the explicit include here also unless we really need it?

Reply:
I've removed both the explicit require for `nokogiri` and gemfile dependency, and the tests are still green 👍 Did not check the `gemspec` file and the other `fog` gems.

Reply:
No worries, it probably wouldn't hurt anything at the moment. But could come back to complicate things later, so probably better to fallback to the existing dependency to keep things simple. Thanks.